tests/contest/runtimetest/src/tests.rs

```diff
@@ -376,3 +380,54 @@ pub fn validate_scheduler_policy(spec: &Spec) {
         eprintln!("error due to sched_nice want {want_sn}, got {sn}")
     }
 }
+
+pub fn test_io_priority_class(spec: &Spec, io_priority_class: IOPriorityClass) {
+    let io_priority_spec = spec
+        .process()
+        .as_ref()
+        .unwrap()
+        .io_priority()
+        .as_ref()
+        .unwrap();
+    if io_priority_spec.class() != io_priority_class {
+        let io_class = io_priority_spec.class();
+        return eprintln!("error io_priority class want {io_priority_class:?}, got {io_class:?}");
+    }
+
+    let io_priority_who_progress: libc::c_int = 1;
+    let io_priority_who_pid = 0;
+    let res = unsafe {
+        libc::syscall(
+            libc::SYS_ioprio_get,
+            io_priority_who_progress,
+            io_priority_who_pid,
+        )
+    };
+    if let Err(e) = Errno::result(res) {
+        return eprintln!("error ioprio_get error {e}");
+    }
+
+    // ref: https://docs.kernel.org/block/ioprio.html
+    let class = res as u16 >> 13;
+    let priority = res as u16 & 0xFF;
+
+    let expected_class = match io_priority_class {
+        IoprioClassRt => 1,
+        IoprioClassBe => 2,
+        IoprioClassIdle => 3,
+    };
+    if class != expected_class {
+        return eprintln!(
+            "error ioprio_get class expected {io_priority_class:?} ({expected_class}), got {class}"
+        );
+    }
+
+    let expected_priority = match io_priority_class {
+        IoprioClassRt => 1,
+        IoprioClassBe => 2,
+        IoprioClassIdle => 3,
+    };
+    if priority != expected_priority {
+        eprintln!("error ioprio_get priority expected {io_priority_class:?} ({priority:?}), got {priority}")
+    }
```

Comment:
For this can you :
1. add a comment above saying these number mappings are arbitrary and done as how we set the priority in test cases
2. the error should not mention the class as that is not really a part of this test, a better error message would be something like `error ioprio_get expected priority {priority:?}, got {priority}`